### ISSUE TITLE
Remove cache_quota limitations

### ISSUE THREAD

**Issue by saelbrec** [openvstorage/framework#1671]
A combination of larger vdisks and a much larger distributed cache than in the past require the ability to configure much larger cache capacities than the currently allowed upper limit of 1TB.

We do not want to enforce these as we don't know all the different use cases/combinations. It's up to the user to make sure he manages and configures this correctly.
So please remove the current implemented limitations for cache quota(both fragment and block).

**Comment by khenderick** [comment]
```
Read / Write on backend (preset "123") 
Cache quota: 43.22 PiB
```
:thumbsup:

**Comment by khenderick** [comment]
Fixed by #1674, packaged in openvstorage-2.9.6-dev.1497355148.5d1277f

**Comment by JeffreyDevloo** [comment]
Packaged in: openvstorage_2.9.6-1_amd64.deb
